playbooks/de-populate-consul.yaml

```diff
@@ -0,0 +1,26 @@
+---
+
+- name: "Update keys in consul"
```

Comment:
Alright. I'll see if that's hard to do (I don't think it is, but I'll always leave the option of me not knowing open :P) and include it if not.